app/serializers/user.js

```diff
@@ -5,9 +5,13 @@ export default ApplicationSerializer.extend({
   serialize(snapshot, options) {
     const json = this._super(...arguments);
     if (snapshot.id) {
-      let attributesToOmit = ['email'];
+      let attributesToOmit = [];
       if (!snapshot.adapterOptions || !snapshot.adapterOptions.includePassword) {
         attributesToOmit.push('password');
+      } else if (!snapshot.adapterOptions || !snapshot.adapterOptions.includeEmail) {
```

Comment:
@niranjan94 If we just keep the first condition ovf includePassword, how would we pick the required fields? It throws error saying 'Missing data for required field.' since all other attributes will also be there in the request body